src/date-range-picker/calendar/header/index.tsx

```diff
@@ -12,7 +12,6 @@ interface CalendarHeaderProps {
   onChangeMonth: (prev?: boolean) => void;
   previousMonthLabel: string;
   nextMonthLabel: string;
-  calendarHasFocus: boolean;
```

Comment:
It would make sense to extract the refactorings that are applicable to the original components to separate PRs.

Reply:
That includes calendarHasFocus and FocusLock changes

Reply:
Separate PR: https://github.com/cloudscape-design/components/pull/121/checks

Reply:
Focus lock PR: https://github.com/cloudscape-design/components/pull/122